modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,697 @@
+= TinyMCE 6.3
+:navtitle: TinyMCE 6.3
+:description: Release notes for TinyMCE 6.3
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.3 was released for {enterpriseversion} and {cloudname} on Wednesday, December 7^th^, 2022. It includes {productname} 6.3, additional changes to Premium plugins, and several new Premium plugins. These release notes provide an overview of the changes for {productname} 6.3:
+
+* xref:new-premium-plugins[New Premium plugins]
+* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:improvements[Improvements]
+* xref:additions[Additions]
+* xref:changes[Changes]
+* xref:bug-fixes[Bug fixes]
+* xref:security-fixes[Security fixes]
+* xref:known-issues[Known issues]
+
+[[new-premium-plugins]]
+== New Premium plugins
+
+=== Inline CSS 1.0.0
+
+The {productname} 6.3 release includes the new **Inline CSS** premium plugin.
+
+The Inline CSS plugin allows the user to <notes here>.
+
+For information on the **Inline CSS** plugin, see: xref:inline-css.adoc[Inline CSS].
+
+=== Advanced Typography 1.0.0
+
+The {productname} 6.3 release includes the new **Advanced typography** premium plugin.
+
+The **Advanced Typography** plugin, depending on its setting, can automatically or interactively replace common typewriter-style characters with their professional typesetter’s equivalents.
```

Comment:
I don't think automatic replacement is a current feature of `typography`

Reply:
@Skylite73
@brian-forte 
Suggested changes:

The **Advanced Typography** plugin, depending on its setting, can replace common typewriter-style characters with their professional typesetter equivalents after the `Advanced Typography` button is pressed.   This action will modify either the entire content for the {productname} document or the specific text selection highlighted by the user.